shuffle/cli/src/node.rs

```diff
@@ -1,27 +1,39 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::new::Config;
 use anyhow::Result;
 use diem_config::config::NodeConfig;
+use diem_genesis_tool::validator_builder::ValidatorConfig;
+use diem_types::{on_chain_config::VMPublishingOption};
+use dir::home_dir;
 use std::{fs, path::Path};
 
-pub fn handle(project_path: &Path) -> Result<()> {
+pub fn handle() -> Result<()> {
     // TODO: Generate prefunded accounts
-    let node_config = NodeConfig::load(project_path.join("nodeconfig/0/node.yaml").as_path())?;
-    let config = read_config(project_path)?;
+    let node_config_path = &(home_dir().unwrap().join(".shuffle"));
```

Comment:
How does this work if there are multiple nodes defined in the shuffle directory? i.e. one for trove local, one for dpn local, etc

Reply:
As discussed offline, we're holding off on specifying multiple nodes bc we don't have separate node configs (dpn, trove) yet

Reply:
You reference this folder in multiple commands. Makes sense to place it in a `shared.rs` file.